README.md

```diff
@@ -1,141 +1,133 @@
-# Strapi intégration
+# MeiliSearch in Strapi Plugin 🔎
```

Comment:
This is ok for the moment, but keep in mind it should have the same header as the other at the end :) I'm going to ask for a MS logo with Strapi :D

Reply:
https://github.com/meilisearch/strapi-plugin-meilisearch/issues/23